components/item_layout/item_layout_default.story.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <dt-item-layout>
+    <template #left>
+      <span v-html="left" />
+    </template>
+    <span v-html="defaultSlot" />
+    <template #subtitle>
+      <span v-html="subtitle" />
+    </template>
+    <template #bottom>
+      <span v-html="bottom" />
+    </template>
+    <template #right>
+      <span v-html="right" />
+    </template>
+    <template #selected>
+      <span v-html="selected" />
+    </template>
+  </dt-item-layout>
```

Comment:
I think you forgot to update these to `$attrs` the component is not currently showing up.

Reply:
> I think you forgot to update these to `$attrs` the component is not currently showing up.

Ah right, thought it was too easy 😅 good catch